openshift/helpers.sh

```diff
@@ -0,0 +1,116 @@
+
+function is_set_or_fail() {
+    local name=$1
+    local value=$2
+
+    if [ -v $value ] || [ $value == "not-set" ]; then
```

Comment:
There's no `$` when using the `-v` check, see `man bash`:

```
-v varname
              True if the shell variable varname is set (has been assigned a value).
```

test yourself:

```
# [ -v BASH ] && echo set || echo not-set
set
# [ -v $BASH ] && echo set || echo not-set
not-set
```

Reply:
Right, good catch. Not the first time I made this mistake.